dask/array/tests/test_fft.py

```diff
@@ -73,6 +126,21 @@ def test_irfft():
     assert_eq(irfft(darr2, axis=0), np.fft.irfft(nparr, axis=0))
 
 
+@pytest.mark.parametrize(
+    "funcname",
+    ["irfft2", "irfftn"]
+)
+def test_irfft2n(funcname):
+    da_fft = getattr(dask.array.fft, funcname)
+    np_fft = getattr(np.fft, funcname)
+    assert_eq(da_fft(darr3),
+              np_fft(nparr))
+    assert_eq(da_fft(darr3, axes=(1, 0)),
+              np_fft(nparr, axes=(1, 0)))
+    assert_eq(da_fft(darr3, (8, 9), axes=(1, 0)),
+              np_fft(nparr, (8, 9), axes=(1, 0)))
```

Comment:
This will probably amount to just writing it, but yeah I can give it some thought. In the interim, I think you are right that we should consolidate these. We can always add new tests for other cases that don't fit the existing motif.